recipes/hgdl/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "hgdl" %}
+{% set version = "2.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lbl-camera/HGDL/archive/{{ version }}.tar.gz
+  sha256: 288ec83fdc95fff3e588a5fb99ead4508138e35407bde70d3a91d0078324792f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - wheel
+    - versioneer
```

Comment:
I see wheel and versioneer in the run deps for these packages but I believe those are not needed Maybe upstream added them by mistake?

Reply:
Versioneer is used in the `setup.py` script. https://github.com/lbl-camera/fvGP/blob/master/setup.py

Reply:
@ocefpaf I added the required patch to remove it from the dependencies.